app/models/liabilitycalculation/AllowancesAndDeductions.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.liabilitycalculation
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class AllowancesAndDeductions(
+                                    personalAllowance: Option[Int] = None,
+                                    marriageAllowanceTransferOut: Option[MarriageAllowanceTransferOut] = None,
+                                    reducedPersonalAllowance: Option[Int] = None,
+                                    giftOfInvestmentsAndPropertyToCharity: Option[Int] = None,
+                                    lossesAppliedToGeneralIncome: Option[Int] = None,
```

Comment:
According to Swagger, there should be a field: blindPersonsAllowance.
If it is not needed it can be ignored.

Reply:
we don't need this field, so not included..